### PR TITLE
Dev jirayu

### PR DESCRIPTION
**Pull request by ChanAndKlee**

### CODE REVIEW

File: lab-exercise-1/JirayuFeature.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+  public class JirayuFeature {
+      public void feature() {
+          // this is a new feature
```
Comment (ChanAndKlee) [inline lab-exercise-1/JirayuFeature.java:3]: This function is incomplete.

**Review by ChanAndKlee — COMMENTED** [review]
Please check my review

**Review by ChanAndKlee — COMMENTED** [review]
Check the function again

File: lab-exercise-1/JirayuFeature.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (ChanAndKlee) [reply lab-exercise-1/JirayuFeature.java:3]: This function is incomplete.

[the remark this answers is not in the document]
Comment (Jack5hj) [reply lab-exercise-1/JirayuFeature.java:3]: this function is need improve

[the remark this answers is not in the document]
Comment (Pontuspts) [reply lab-exercise-1/JirayuFeature.java:3]: This content isn't well-written, please improve it.

[the remark this answers is not in the document]
Comment (KRoos555) [reply lab-exercise-1/JirayuFeature.java:3]: This contents is not well-written. Please improve it.

**Review by NoodlesCupCake — COMMENTED** [review]
Fixed now

**Review by Pontuspts — COMMENTED** [review]
good

File: lab-exercise-1/JirayuFeature.java [hunk 2]
```diff
@@ -1,5 +1,7 @@
   public class JirayuFeature {
       public void feature() {
           // this is a new feature
+          int x = 0;
+          x = x+1;
```
Comment (NoodlesCupCake) [inline lab-exercise-1/JirayuFeature.java:5]: good James

Comment (Vvidsky) [inline lab-exercise-1/JirayuFeature.java:5]: Nice one!

**Review by Vvidsky — APPROVED** [review]

File: lab-exercise-1/JirayuFeature.java [hunk 3]
```diff
@@ -0,0 +1,7 @@
+  public class JirayuFeature {
+      public void feature() {
+          // this is a new feature
+          int x = 0;
```
Comment (ChanAndKlee) [inline lab-exercise-1/JirayuFeature.java:4]: Good James!

**Review by ChanAndKlee — COMMENTED** [review]
👍🏻

File: lab-exercise-1/JirayuFeature.java [hunk 4]
```diff
@@ -0,0 +1,7 @@
+  public class JirayuFeature {
+      public void feature() {
+          // this is a new feature
```
Comment (ChanAndKlee) [inline lab-exercise-1/JirayuFeature.java:3]: Good job

**Pull request merged**